localstack/services/awslambda/lambda_starter.py

```diff
@@ -48,3 +63,33 @@ def check_lambda(expect_shutdown=False, print_error=False):
         assert out is None
     else:
         assert out and isinstance(out.get("Functions"), list)
+
+
+def apply_patches():
```

Comment:
nitpick: Wrapping the patches in a function is technically no longer required - the new plugin loading mechanism takes care that the code in this file is lazily loaded (i.e., only loaded when Lambda API is first requested). But can't harm to have it in here for now - we'll likely perform some major refactoring of the Lambda API anyway.. 👍